.travis-setup.sh

```diff
@@ -35,42 +35,46 @@ sudo apt-get -qq install valgrind lcov uuid-dev pkg-config \
   zlib1g-dev libffi-dev gettext libpcre3-dev cppcheck \
   libmnl-dev
 
+function compile {
+	name=$1
+	tarball=$2
+	directory=$3
```

Comment:
I'd personally always quote variables like this to avoid potential problems in the future wrt spaces. It's unlikely ever to be a problem under Travis with this particular function, but just good practise.